docs/conf.py

```diff
@@ -336,6 +346,9 @@
     },
 }
 
+# build needs.json to make permalinks work
+needs_build_json = True
```

Comment:
@danwos I activated this to make permalinks work in the needs docs.

Reply:
makes sense